python/exercise2/time_window.py

```diff
@@ -4,10 +4,10 @@ class TimeWindow:
     """ Observations for a TIEGCM experiment """
     
     def __init__(self, start_time, end_time, delta):
-        """ Initializes the observations associated with a TIEGCM experiment
+        """ Initialize the window and model times associated with a TIEGCM experiment
         
-        start_time - string '%Y-%m-%d %H:%M:%S' model start time of experiment
-        end_time   - string '%Y-%m-%d %H:%M:%S' model end time of experiment
+        start_time - string '%Y-%m-%d %H:%M:%S' model inital time
```

Comment:
I would recommend keeping the phrasing of the definition to be 'start' instead of 'initial' to be consistent with the variable name.